sdk/src/event/async_timer/dispatcher.cc

```diff
@@ -9,15 +13,29 @@ namespace async_timer
 {
 std::unique_ptr<event::Timer> Dispatcher::CreateTimer(TimerCallback callback) noexcept
 {
-  (void)callback;
-  return nullptr;
+  return std::unique_ptr<event::Timer>{new (std::nothrow) Timer{callback, *this}};
 }
 
-void Dispatcher::Exit() noexcept {}
+void Dispatcher::Exit() noexcept
+{
+  running_ = false;
```

Comment:
Not meant to be threadsafe?

Reply:
Yes, meant for interacting with only a single background thread.